brvs-core/src/main/java/iroha/validation/utils/ValidationUtils.java

```diff
@@ -12,4 +16,18 @@ static String getTxAccountId(final Transaction transaction) {
   static String hexHash(Transaction transaction) {
     return Utils.toHex(Utils.hash(transaction));
   }
+
+  static KeyPair readKeyPairFromFiles(String pubKeyPath, String privKeyPath) {
```

Comment:
What if arguments are `null`?

Reply:
Are you just kidding? You will get an exception on startup :)